back/spec/factories/topics.rb

```diff
@@ -1,39 +1,37 @@
 FactoryBot.define do
   factory :topic do
-    title_multiloc {{
-      "en" => "Health",
-      "nl-BE" => "Gezondheid"
-    }}
-    description_multiloc {{
-      "en" => "<p>How are the people feeling?</p><p>What is the expected agespan?</p>",
-      "nl-BE" => "<p>Hoe voelen de mensen zich?</p><p>Wat is de levensverwachting?</p>"
-    }}
+    transient do
```

Comment:
I reworked the topic factory to make it easier to create topics with different titles (useful in this case since categories cannot have the same name).